seqr/management/tests/check_for_new_samples_from_pipeline_tests.py

```diff
@@ -282,10 +287,14 @@ def _test_call(self, error_logs, reload_annotations_logs=None, run_loading_logs=
     @mock.patch('seqr.management.commands.check_for_new_samples_from_pipeline.MAX_LOOKUP_VARIANTS', 1)
     @mock.patch('seqr.views.utils.airtable_utils.BASE_URL', 'https://test-seqr.org/')
     @mock.patch('seqr.views.utils.airtable_utils.MAX_UPDATE_RECORDS', 2)
+    @mock.patch('seqr.views.utils.export_utils.os.makedirs')
+    @mock.patch('seqr.views.utils.export_utils.mv_file_to_gs')
```

Comment:
since we are already mocking the subprocess calls used for interacting with `gs` in this test case it would be better not to mock this helper function and instead assert that the subprocess calls all look right including the `mv` calls